cpp/src/Slice/Grammar.y

```diff
@@ -206,11 +206,11 @@ opt_semicolon
 // ----------------------------------------------------------------------
 global_meta_data
 // ----------------------------------------------------------------------
-: ICE_GLOBAL_METADATA_OPEN string_list ICE_GLOBAL_METADATA_CLOSE
+: ICE_FILE_METADATA_OPEN string_list ICE_FILE_METADATA_CLOSE
 {
     $$ = $2;
 }
-| ICE_GLOBAL_METADATA_IGNORE string_list ICE_GLOBAL_METADATA_CLOSE
+| ICE_FILE_METADATA_IGNORE string_list ICE_FILE_METADATA_CLOSE
 {
     unit->error("global metadata must appear before any definitions");
```

Comment:
Should you fix the message as well?

Reply:
Yes, you're correct about all these, I'll correct the naming for them as well.